### PR TITLE
add scroll in editor window

### PR DESCRIPTION
**Pull request by Lido233**

### CODE REVIEW

File: com.unity.uiwidgets/Editor/UIWidgetsEditorPanel.cs [hunk 1]
```diff
@@ -108,6 +108,13 @@ void OnGUI()
                 var pos = _getPointerPosition(evt.mousePosition);
                 _wrapper.OnMouseMove(pos);
             }
+            else if (evt.type == EventType.ScrollWheel && evt.delta.magnitude != 0) {
+                var delta = evt.delta;
+                delta.x  = 0f;
```
Comment (zhuxingwei) [inline com.unity.uiwidgets/Editor/UIWidgetsEditorPanel.cs:113]: why making it 0? we need accept and process non-zero delta x too

File: com.unity.uiwidgets/Runtime/rendering/list_wheel_viewport.cs [hunk 2]
```diff
@@ -571,7 +571,7 @@ public class RenderListWheelViewport : ContainerRenderObjectMixinRenderBox<Rende
             var deltaY = radius * Mathf.Sin(angle);
 
             Matrix4 transform = MatrixUtils.createCylindricalProjectionTransform(
-                radius: size.height * _diameterRatio / 2.0f,
+                radius: size.height * _diameterRatio / 2.0F,
```
Comment (zhuxingwei) [inline com.unity.uiwidgets/Runtime/rendering/list_wheel_viewport.cs:574]: f is ok, no need to change to F?

File: com.unity.uiwidgets/Editor/UIWidgetsEditorPanel.cs (hunk 1 above)
Comment (Lido233) [reply com.unity.uiwidgets/Editor/UIWidgetsEditorPanel.cs:113]: ok, I'll fix it.

File: com.unity.uiwidgets/Runtime/rendering/list_wheel_viewport.cs (hunk 2 above)
Comment (Lido233) [reply com.unity.uiwidgets/Runtime/rendering/list_wheel_viewport.cs:574]: ok, I'll fix it.

**Pull request merged**